test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -65,13 +70,68 @@ def _parameterize_class_name(cls: Type, idx: int, input_dicts: Mapping[Any, Any]
     return f"{cls.__name__}_{'_'.join(suffixes)}"
 
 
+def skip_model_type_is_exported_program_test(reason: str):
```

Comment:
What is the reason we don't put them into pytorch_test_common.py?

Reply:
circular import dependency issue between `onnx_test_common.py` and `pytorch_test_common.py` files. To move `skip_model_type_is_exported_program_test` inside `pytorch_test_common`, we would have to do `import onnx_test_common` inside `skip_model_type_is_exported_program_test` function instead of the beginning of `pytorch_test_common.py` as usual.

I used the same approach as [test_fx_op_consistency.py](https://github.com/pytorch/pytorch/blob/16376af6766169c9cefd2baf48c344dacae83d34/test/onnx/test_fx_op_consistency.py#L171) that defined its own `xfail_torchlib_forward_compatibility` locally as opposed to where the original `xfail` lived (test/onnx/onnx_test_common.py)

Reply:
moved to pytorch_test_common.py and used xfail instead of skip